sql_export_excel/models/sql_export.py

```diff
@@ -69,15 +72,87 @@ def check_column_position(self):
                     _("The column position can't be less than 1.")
                 )
 
-    @api.multi
     def _get_file_extension(self):
         self.ensure_one()
         if self.file_format == "excel":
             return "xlsx"
         else:
             return super()._get_file_extension()
 
-    @api.multi
+    # API Section
+    def _execute_sql_request(
```

Comment:
You can't just copy past the whole method.
The header modification should be done in base module ( `sql_request_abstract`) also because it could be re-used by another module.
Maybe I was not specific enough, I think you can do the header modification in this PR, but it should be done in `sql_request_abstract` directly, the same way it was done in v12, in file  sql_request_abstract/models/sql_request_mixin.py (https://github.com/OCA/server-tools/pull/1659/files).

Reply:
The suggestion is implemented can you please re-review it